pom.xml

```diff
@@ -19,8 +19,8 @@
   </modules>
 
   <properties>
-    <raml-module-builder.version>33.2.2</raml-module-builder.version>
-    <vertx.version>4.2.1</vertx.version>
+    <raml-module-builder.version>33.2.6</raml-module-builder.version>
+    <vertx.version>4.2.5</vertx.version>
     <junit.version>4.13</junit.version>
```

Comment:
On standup we agreed just to override log4j version to 2.17 in the pom file. Can you attach screen with vulnerabilities before/after Vertx upgrade

Reply:
Before rmb and vertx upgrade:
![image](https://user-images.githubusercontent.com/89521577/155307603-1d7e00e1-e765-4f54-9c7f-a819779007c0.png)
After:
![image](https://user-images.githubusercontent.com/89521577/155307824-ff61d913-8bb8-45b4-b4eb-23e15c20ceef.png)
As result we can see four fixed vulnarabilities.

Reply:
Thanks for screens, so l propose just to fix High critical dependency vulnerability with log4j by overriding it version in Maven, because of code freeze and risks connected with upgrading RMB/Vertx before release, and create additional follow-up story to upgrade RMB/Vertx for this module in Morning Glory

Reply:
Totally agree. Let's postpone RMB/Vertx until Morning Glory.